llvm-project/stage3.cmake

```diff
@@ -0,0 +1,67 @@
+# Please try to keep these cmake variables alphabetically sorted.
```

Comment:
(github code review is broken. To me it looks like there is two threads about this topic, but the "Files changed" tab shows only this one.)

Reply:
and your reply to @compnerd isn't threaded properly, WTF is this broken-@$$ code review system...JFC

Reply:
> (github code review is broken. To me it looks like there is two threads about this topic, but the "Files changed" tab shows only this one.)

Weird…

> and your reply to @compnerd isn't threaded properly, WTF is this broken-@$$ code review system...JFC

This one is on me, I manually quoted and replied because further down, this comment is not quotable…

Reply:
Oh, I see what happened… I started a review before you deleted those lines in a follow up commit so GitHub cannot show my first comment with the current diff. Sigh…

Reply:
that makes sense; still, poor UI

Reply:
I can probably just add them to be explicit, even though they are implicit.

I was thinking more about refactoring/including cmake files. Could probably have the shared base file set things we ultimately want, then children/includers reset new values when the desired values can't be used yet.

Thoughts?

Reply:
That seems sound to me but I am not too familiar with CMake to say if that is the proper way to go about things (as opposed to only factoring out common variables).

Reply:
done PTAL